### ISSUE TITLE
Defocus/Refocus should maintain focus

### ISSUE THREAD

**Issue by robomac** [WordPress/gutenberg#13146]
If I'm in a tabbed browser, writing a "block" of text, and tab (<ctrl-tab> or click on another tab) over to get, e.g., the name or link... Gutenberg loses the focus.  Sometimes it stays, sometimes it goes to the front of the text block and sometimes it's not on any editable.  

When I <ctrl-tab> back and type (or paste), the previous point should still be active.  Always.  Not just sometimes.  This was not a problem with the previous editor.

**Desktop (please complete the following information):**
 - Windows 10
 - Chrome 
 - Version 71.0.3578.98

**Comment by afercia** [comment]
@robomac thanks for your report. I'm not able to reproduce. Could you please provide detailed steps to reproduce the issue?

Does it still happen with all plugins disabled and a default theme (for example Twenty Nineteen or Twenty Seventeen) activated?

**Comment by noisysocks** [comment]
_This issue was [raised](https://wordpress.slack.com/archives/C02QB2JS7/p1579757176230200) in a #core-editor triage session._

I wasn't able to reproduce this in the development version of WordPress + Gutenberg. Closing this out for now, but we can re-open if there's more info.
